<launch>
    <node name="turtlesim" pkg="turtlesim" type="turtlesim_node" />
    <node name="joy_node" pkg="joy" type="joy_node"/>
    <node name="Joystick" pkg="joystick" type="joystick_node" output="screen"/>

        <param name="num_axe_lin" value="1" type="int"/>
        <param name="num_axe_ang" value="2" type="int"/>
        <param name="factor_axe_lin" value="1.0" type="double"/>
        <param name="factor_axe_ang" value="1.0" type="double"/>
        <param name="pub_name" value="turtle1/cmd_vel" type="string"/>

</launch>
